setuptools/dist.py

```diff
@@ -83,6 +84,24 @@ def _read_list(name):
             return None
         return values
 
+    def _read_long_description():
+        value = msg['description']
+        if value in ('UNKNOWN', None):
```

Comment:
I'm uncomfortable with these "UNKNOWN" literals. Does Setuptools add "UNKNOWN" for unsupplied long descriptions?

Reply:
`UNKNOWN` is the `distutils` default for `get_long_description()`. Since this is used when writing the metadata file, either the header field or the payload will contain `UNKNOWN` in case no `description` is given.
https://github.com/pypa/setuptools/blob/03ad13c97fc47b4773c49fa486f7f9b3349b7e56/setuptools/_distutils/dist.py#L1197-L1198